index.js

```diff
@@ -160,40 +152,31 @@ const queryHttps = (version, options) => {
 
 			const urls = [].concat.apply(type[version].httpsUrls, options.fallbackUrls || []);
 
-			for (const url of urls) {
+			// Make object with url as key and pattern as value
+			const urls_ = {
+				...urls.reduce((result, item) => {
```

Comment:
Don't use `reduce`. It makes code harder to read. A `for-of` loop is more readable.